example/cloud-config.yml

```diff
@@ -0,0 +1,51 @@
+#######################
+## Availabilty Zones ##
+#######################
+
+azs:
+- name: z1
+
+##############
+## VM Types ##
+##############
+
+vm_types:
+- name: default
+  cloud_properties:
+    ephemeral_disk:
+      size: 1024
+      type: gp2
+
+################
+## Disk Types ##
+################
+
+disk_types:
+- name: default
+  disk_size: 1024
+
+#################
+## Compilation ##
+#################
+
+compilation:
+  workers: 2
+  network: default
+  az: z1
+  reuse_compilation_vms: false
+  vm_type: default
+
+##############
+## Networks ##
+##############
+
+networks:
+- name: default
+  type: manual
+  subnets:
+  - range: 10.244.1.0/24
```

Comment:
I would pick something other than 10.244.1. The cf-infrastructure-bosh-lite already has 10.244.1 defined so we could collide.